digicampipe/scripts/raw.py

```diff
@@ -9,12 +9,16 @@
   -h --help                   Show this screen.
   --max_events=N              Maximum number of events to analyse
   -o FILE --output=FILE.      File where to store the results.
-                              [Default: .]
-  -c --compute                Compute the data.
+                              [Default: ./raw_histo.pk]
+  -c --compute                Compute the raw data histograms.
   -d --display                Display.
   -p --pixel=<PIXEL>          Give a list of pixel IDs.
   --save_figures              Save the plots to the same folder as output file.
-'''
+  --baseline_filename=FILE    Output path for baseline histogram. If None the
```

Comment:
It is not clear what baseline histogram it is. I would suggest something like "Output path for DigiCam calculated baseline histogram."